### ISSUE TITLE
Cell(0 0) in the table would be better like SRFI/Implementation platform

### ISSUE THREAD

**Issue by APIPLM** [schemedoc/srfi-metadata#38]
Cell(0 0) in the table would be better like SRFI/Implementation platform.

**Comment by jpellegrini** [comment]
Sure! But see #19  -- the appearance of the table needs a lot of work.

**Comment by lassik** [comment]
Yes, it would be great to have a graphic designed. I've been trying to reach David by mail but haven't been able to.

**Comment by APIPLM** [comment]
Graphic designed is needed. But what I mean is that it is a table, it would have a group name of the columns, Is that name "Implementation platform" precise. As we look at the each implementation, sound like they all have their purposes or new strong features, like "Chibi-Scheme is a very small library intended for use as an extension and scripting language in C programs. In addition to support for lightweight VM-based threads, each VM itself runs in an isolated heap allowing multiple VMs to run simultaneously in different OS threads"

**Comment by lassik** [comment]
_Scheme implementation_ would be the precise term. But it's probably clear from their names that they are Scheme implementations?

**Comment by APIPLM** [comment]
Yes. you are right. It is all about SRFIs implementations More focus on which SRFI in which Scheme implementation. I am trying to figure out that SRFIs of the author are closer to which role of person. Program Language Scheme Designer Like R5RS, R6RS R7RS or Features Designer Scheme or Implementor of Scheme Features. Seems like it is a combination.

**Comment by lassik** [comment]
The people listed as "author" either wrote the text of the SRFI document, or wrote the code of the sample implementation, or both. It is usually a combination of both.
